engine/script/src/script_module.cpp

```diff
@@ -142,7 +102,7 @@ namespace dmScript
             return 1;
         }
 
-        if (!LuaLoadModule(L, module->m_Script, module->m_ScriptSize, name))
+        if (!LuaLoadModule(L, module->m_Script, module->m_ScriptSize, module->m_Filename))
```

Comment:
We do the same for Lua modules. Always use what LuaBuilder.java has provided as filename.